### ISSUE TITLE
Event Templates with multi-word Labels cannot be used

### ISSUE THREAD

**Issue by andrewazores** [cryostatio/cryostat#890]
Steps to reproduce:

1. Open web-client
2. Navigate to Events
3. Download Profiling event template
4. Edit Profiling event template and change `label` attribute to ex. `Broken Template`, or some other multi-word string, save file as `Broken.jfc`
5. Click Upload button in web-client
6. Select and upload `Broken.jfc`
7. "Broken Template" appears in Event Templates list, as expected
8. Click "Broken Template" overflow menu and select "Create Recording..."
9. Enter a recording name and try to start the new recording
10. Observe recording start failure due to `400 Bad Request / IllegalArgumentException`

Expected behaviour:
Step 10 should succeed, the recording should be created despite the template having a multi-word label/name.

**Comment by andrewazores** [comment]
```
WARNING: HTTP 400: template=Failure Template,type=CUSTOM
io.vertx.ext.web.handler.impl.HttpStatusException: Bad Request
Caused by: java.lang.IllegalArgumentException: template=Failure Template,type=CUSTOM
	at io.cryostat.recordings.RecordingTargetHelper.parseEventSpecifierToTemplate(RecordingTargetHelper.java:437)
	at io.cryostat.net.web.http.api.v1.TargetRecordingsPostHandler.lambda$handleAuthenticated$0(TargetRecordingsPostHandler.java:200)
	at io.cryostat.net.TargetConnectionManager.executeConnectedTask(TargetConnectionManager.java:148)
	at io.cryostat.net.TargetConnectionManager.executeConnectedTask(TargetConnectionManager.java:126)
	at io.cryostat.net.web.http.api.v1.TargetRecordingsPostHandler.handleAuthenticated(TargetRecordingsPostHandler.java:160)
	at io.cryostat.net.web.http.AbstractAuthenticatedRequestHandler.handle(AbstractAuthenticatedRequestHandler.java:99)
	at io.cryostat.net.web.http.AbstractAuthenticatedRequestHandler.handle(AbstractAuthenticatedRequestHandler.java:69)
	at io.vertx.ext.web.impl.BlockingHandlerDecorator.lambda$handle$0(BlockingHandlerDecorator.java:48)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$2(ContextImpl.java:313)
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by andrewazores** [comment]
Closed by #898
